deletefb/tools/chrome_driver.py

```diff
@@ -0,0 +1,111 @@
+import re
+import zipfile
+import os, sys, stat, platform
+from urllib.request import urlretrieve
+
+from clint.textui import puts, colored
+import progressbar
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+from .common import NO_CHROME_DRIVER
+
+
+def extract_zip(filename):
+    """
+    Uses zipfile package to extract a single zipfile
+    :param filename:
+    :return: new filename
+    """
+    try:
+        file = zipfile.ZipFile(filename, 'r')
+    except FileNotFoundError:
+        puts(colored.red(f"{filename} Does not exist"))
+        sys.exit()
```

Comment:
I like to pass an exit code when calling `sys.exit()`, so e.g. `sys.exit(1)` just to be more explicit about the error codes